File: tests/fixtures/corpus/src/go/cases.go
package corpus

import "errors"

func structured() error {
	return NewAppError("GO_USER_MISSING", "User was not found", 404)
}

func dynamic(dynamicCode string) error {
	return NewAppError(dynamicCode, "Dynamic code remains unresolved", 500)
}

func generic() error {
	return errors.New("Invalid state")
}

var ignored = errors.New("GO_NOISE")

File: tests/fixtures/corpus/src/swift/Cases.swift
func structured() throws {
  throw APIError.failure("SWIFT_USER_MISSING", "User was not found", 404)
}

func dynamic(_ dynamicCode: String) throws {
  throw APIError.failure(dynamicCode, "Dynamic code remains unresolved", 500)
}

func generic() throws {
  throw NetworkError.offline
}

let ignored = NetworkError.offline
